students/17th/mikyung/js/login.js

```diff
@@ -0,0 +1,16 @@
+const thisIsButton = document.getElementById('b0tn_login')
+
+thisIsButton.addEventListener('click' function() {
+    const id = document.getElementById('userID').value;
+    const password = document.getElementById('userPW').value;
+
+    if (!id) {
+        alert('전화번호, 사용자 이름 또는 이메일을 입력해주세요!')
+        return;
+
+        if (!password) {
+            alert('비밀번호를 입력해주세요!')
+returnl}
+alert('로그인 성공!')
+    }
+}
```

Comment:
작성해주신 코드를 기반으로 몇가지 코멘트 드리겠습니다.
```suggestion
const loginBtn = document.getElementById('b0tn_login')

loginBtn.addEventListener('click' function() {
    const id = document.getElementById('userID').value;
    const password = document.getElementById('userPW').value;

    if (!id) {
       alert('전화번호, 사용자 이름 또는 이메일을 입력해주세요!')
       return;
    }

    if (!password) {
      alert('비밀번호를 입력해주세요!')
      return;
    }

    alert('로그인 성공!')
}
```
- 개발자는 코드를 기반으로 소통해야 하기 때문에 어떤 것이 읽기 좋은 코드일지 많이 고민해보셔야 합니다. (물론 처음부터 그렇게 되는 것은 아닐 것이고 앞으로 천천히 개선해나가시면 됩니다!)
- 읽기 좋은 코드의 기준으로 여러가지가 있겠지만 그중 가장 먼저 챙겨야할 부분이 바로 띄어쓰기, 변수명 등의 컨벤션입니다.
- 이런걸 매번 번거롭게 하기 어려우니 eslint, prettier 같은 툴들이 유용하게 쓰입니다. 한번 검색해보세요!
- `thisIsButton`보다는 `button`, `btn`이나, 어떤 버튼인지 알려주는 `loginBtn` 등의 이름이 더 좋을 것 같습니다.
- `returnl` 오타입니다 => `return;`
- `id`가 없을 때 => 추가로 `password`를 체크하고 이것들이 모두 아닐 때 로그인 성공을 출력해야 하고 있네요! 잘 작성해주셨는데, if문이 중첩되는 건 가독성 측면에서 가급적 피해야 할 패턴입니다. 같은 depth에서 return과 함께 써주면 동일한 동작을 하게됩니다. 코드 확인해주세요.